elasticsearch/search-index-settings.json

```diff
@@ -268,6 +268,13 @@
         "type":"text",
         "analyzer":"ons_standard"
       },
+      "canonical_topic":{
```

Comment:
Suggest ordering fields in model alphabetically to more easily find fields (when we want to check a field exists in the model it will be quicker to identify)

Reply:
This is not ordered at all. I have placed the canonical_topic close to the cancelled. Do you mean to order all the fields in the file ?

Reply:
Yes